docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -51,31 +54,64 @@
 
 \pagenumbering{arabic}
 
-\wss{You are free to modify this template.}
-
 \section{Introduction}
 
-\wss{You can include your definition of what a hazard is here.}
+A hazard in the context of this document is any property or condition that may
```

Comment:
wondering if we should say anything that would cause a loss in terms of performance or accuracy?

Reply:
ooh yeah sure I'll add that

Reply:
actually I moved the defn from scope to here, I think it flows better that way